tensorrt_llm/_torch/custom_ops/cute_dsl_custom_ops.py

```diff
@@ -1311,19 +1312,23 @@ def forward(self, inputs: List[torch.Tensor],
             stream = cuda.CUstream(torch_stream.cuda_stream)
 
             if isinstance(tactic, tuple):
-                mma_tiler_mn, cluster_shape_mn = tactic
+                mma_tiler_mn, cluster_shape_mn, raster_along_m = tactic
             else:
-                mma_tiler_mn, cluster_shape_mn = (128, 128), (1, 1)
+                if self.tile_size == 256:
+                    mma_tiler_mn, cluster_shape_mn, raster_along_m = (
+                        256, 128), (2, 1), False
+                else:
+                    mma_tiler_mn, cluster_shape_mn, raster_along_m = (
+                        128, 128), (1, 1), False
```

Comment:
Could be simplified as:

```suggestion
                mma_tiler_mn, cluster_shape_mn, raster_along_m = (
                    self.tile_size, 128), (2, 1), False
```